model.js

```diff
@@ -39,3 +45,20 @@ exports.updateReview = (review_id, inc_votes) => {
       return updatedReview;
     });
 };
+
+exports.fetchReviews = (filter) => {
+  let queryString = `SELECT users.username AS owner, reviews.* , COUNT(comments.review_id) comment_count
+FROM reviews
+LEFT JOIN comments ON reviews.review_id = comments.review_id
+LEFT JOIN users ON reviews.owner = users.username`;
+
+  if (filter) {
+    queryString += ` WHERE category = '${filter}'`;
```

Comment:
we should be using parameterised query here rather than string literals `$1` vs `${filter}` to avoid sql injection